examples/pumi_atr_ex2.cpp

```diff
@@ -0,0 +1,427 @@
+//                                MFEM Example 2 modified
```

Comment:
I suggest creating a directory `examples/pumi` and moving `ex1`, `ex1p` and `ex2p` there, following the style of `examples/sundials` and `examples/petsc`.

Reply:
Done.